source/iterators.tex

```diff
@@ -6520,6 +6523,28 @@
 \pnum \returns \tcode{N}.
 \end{itemdescr}
 
+\indexlibrary{\idxcode{ssize(C\& c)}}%
+\begin{itemdecl}
+template <class C>
+constexpr auto ssize(const C& c)
+    -> common_type_t<ptrdiff_t, make_signed_t<decltype(c.size())>>;
+\end{itemdecl}
+\begin{itemdescr}
+\pnum \returns
+\begin{codeblock}
+static_cast<common_type_t<ptrdiff_t, make_signed_t<decltype(c.size())>>>(c.size())
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibrary{\idxcode{ssize(T (\&array)[N])}}%
+\begin{itemdecl}
+template <class T, ptrdiff_t N>
+constexpr ptrdiff_t ssize(const T (&array)[N]) noexcept;
```

Comment:
Place on same line so it appears as it does in the declaration in the synopsis.

Reply:
Fixed.